README.rst

```diff
@@ -17,13 +17,17 @@ API specificaties
 =================
 
 ==========  ==============  =============================
-Versie      Release datum   API specificatie 
+Versie      Release datum   API specificatie
 ==========  ==============  =============================
 1.0.1       2019-12-16      `ReDoc <https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/VNG-Realisatie/gemma-documentregistratiecomponent/1.0.1/src/openapi.yaml>`_,
                             `Swagger <https://petstore.swagger.io/?url=https://raw.githubusercontent.com/VNG-Realisatie/gemma-documentregistratiecomponent/1.0.1/src/openapi.yaml>`_
-                            (`verschillen <https://github.com/VNG-Realisatie/gemma-documentregistratiecomponent/compare/1.0.0...1.0.1?diff=split#diff-b9c28fec6c3f3fa5cff870d24601d6ab7027520f3b084cc767aefd258cb8c40a>`_)
+                            (`verschillen <https://github.com/VNG-Realisatie/gemma-documentregistratiecomponent/compare/1.0.0...1.0.1?diff=split#diff-b9c28fec6c3f3fa5cff870d24601d6ab7027520f3b084cc767aefd258cb8c40a>`_),
+                            `scopes <https://github.com/VNG-Realisatie/gemma-documentregistratiecomponent/blob/1.0.1/src/scopes.md>`_,
+                            `kanalen <https://github.com/VNG-Realisatie/gemma-documentregistratiecomponent/blob/1.0.1/src/kanalen.md>`_
```

Comment:
@joeribekker in het issue had zei je dat deze bij de ondersteuningstabel moesten, maar kunnen ze niet beter hier aangezien ze versieafhankelijk (kunnen) zijn?

Reply:
Ja, je hebt gelijk. Kan je het in 2 kolommen toevoegen?

```
=================  =================
Autorisaties       Notificaties
=================  =================
Scopes             Berichtkenmerken
=================  =================
```